app/models/post.rb

```diff
@@ -192,7 +192,7 @@ def read_time_for(viewing_replies)
 
     # testing for case where the post was changed in status more recently than the last reply
     audits_exist = audits.where('created_at > ?', most_recent.created_at).where(action: 'update')
-    audits_exist = audits_exist.where("(audited_changes -> 'status' ->> 1)::integer = ?", Post::Status::COMPLETE)
+    audits_exist = audits_exist.where("(audited_changes -> 'status' ->> 1)::integer = ?", Post.statuses[:complete])
```

Comment:
This is one of the cases I expect must validly remain calling into the hash for the integer value, given the SQL givens

Reply:
~SQL~